core/shared/src/test/scala/fs2/Fs2Suite.scala

```diff
@@ -58,3 +60,42 @@ abstract class Fs2Suite
     registered.foreach(t => munitTestsBuffer += t.withName(s"$name - ${t.name}"))
   }
 }
+
+trait StreamAssertions { self: CatsEffectSuite =>
+  implicit class StreamAssertionsOf[A](val str: Stream[IO, A]) {
+
+    def assertEmpty(): IO[Unit] =
+      str.compile.toList.assertEquals(Nil)
+
+    def emitsOutputs(expectedOutputs: List[A]): IO[Unit] =
```

Comment:
Style wise, can we start all these with `assert` -- e.g. `assertSameOutputAs(...)`. That way it remains clear where the assertions are being performed.

Reply:
Renamed methods to use assert word. Also, added a method that takes a varargs.

Reply:
Removed that method, it did not compile in Scala 3.